VideoLocker/src/main/java/org/edx/mobile/base/BaseSingleFragmentActivity.java

```diff
@@ -80,6 +81,11 @@ protected void onOffline() {
         super.onOffline();
         showOfflineBar();
         invalidateOptionsMenu();
+
+        //If the Handouts screen goes offline, we need to show Offline message
+        if(singleFragment!=null && singleFragment instanceof CourseHandoutFragment){
```

Comment:
This is a generic base class. It shouldn't need to know about one specific case. Can we put this in CourseHandoutActivity